src/datePicker/index.tsx

```diff
@@ -1,132 +1,194 @@
-import React, { Component } from 'react'
+import React, { PureComponent } from 'react'
 import cc from 'classcat'
-import moment from 'moment'
-import 'react-dates/initialize'
-import { DayPickerSingleDateController, isInclusivelyAfterDay } from 'react-dates'
-
-import {
-  HORIZONTAL_ORIENTATION,
-  VERTICAL_ORIENTATION,
-  VERTICAL_SCROLLABLE,
-} from 'react-dates/constants'
+import DayPicker, { DayModifiers, NavbarElementProps, CaptionElementProps } from 'react-day-picker'
 
+import prefix from '_utils'
 import { color } from '_utils/branding'
 import ArrowIcon from 'icon/arrowIcon'
-import style from './style'
-import theme from './theme'
+import Button from 'button'
+
+import style from 'datePicker/style'
+
+const [BASE_CLASSNAME] = prefix({ datepicker: true })
 
-const navPrev = <ArrowIcon iconColor={color.primary} />
-const navNext = <ArrowIcon right iconColor={color.primary} />
+const defaultWeekdaysLong = [0, 1, 2, 3, 4, 5, 6].map(weekday =>
+  DayPicker.LocaleUtils.formatWeekdayLong(weekday),
+)
+const defaultWeekdaysShort = [0, 1, 2, 3, 4, 5, 6].map(weekday =>
+  DayPicker.LocaleUtils.formatWeekdayShort(weekday),
+)
+const defaultMonths = DayPicker.LocaleUtils.getMonths()
+
+enum Orientation {
```

Comment:
Should probably be exported for external use to prevent clients from duplicating this.

Reply:
It's available as a static with `DatePicker.constants` ;)